lookout/style/format/benchmarks/__main__.py

```diff
@@ -0,0 +1,65 @@
+"""CLI for benchmarking."""
```

Comment:
should be in `format/__main__.py` to have only one entry point to all the supported tools.